pom.xml

```diff
@@ -1514,12 +1514,12 @@
           <excludedLinks>
             <excludedLink>reports/google-style/guava</excludedLink>
             <excludedLink>reports/javadoc/openjdk8</excludedLink>
+            <!-- posting to the mailing list "checkstyle-announce" is private -->
```

Comment:
[maling-lists.html](https://checkstyle.org/mailing-lists.html) contains unclickable link. The `post` element is required, but can't be filled with the right value.